app/bundles/FormBundle/Model/FormModel.php

```diff
@@ -497,24 +497,29 @@ public function getLeadSubmissions(Form $form, $leadId, $limit = 200)
      */
     public function generateHtml(Form $entity, $persist = true)
     {
-        //generate cached HTML
-        $theme       = $entity->getTemplate();
-        $submissions = null;
-        $lead        = ($this->requestStack->getCurrentRequest()) ? $this->contactTracker->getContact() : null;
-        $style       = '';
+        $theme         = '@themes/'.$entity->getTemplate().'/Field/';
```

Comment:
@escopecz Is this the best location to put custom templates inside themes for fields? or do you recommend something else?

Reply:
No, I think it is good this way